From 57db0237f4f2f28a65b9f1100c2e5b4fbd22e0f5 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Tue, 18 Aug 2020 05:58:54 -0700
Subject: [PATCH 1524/1852] v4l: xilinx: xcsi2rxss: Add YUV 420 video format
 code

commit 25d630ab1e902997f2a6ec1bdceec31208412cb6 from
https://github.com/Xilinx/linux-xlnx.git

Add YUV 420 video format code to support YUV 420 8bpc data type.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-csi2rxss.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/platform/xilinx/xilinx-csi2rxss.c b/drivers/media/platform/xilinx/xilinx-csi2rxss.c
index bc4086ed7b28..8f90abb67f3a 100644
--- a/drivers/media/platform/xilinx/xilinx-csi2rxss.c
+++ b/drivers/media/platform/xilinx/xilinx-csi2rxss.c
@@ -1763,6 +1763,7 @@ static int xcsi2rxss_parse_of(struct xcsi2rxss_state *xcsi2rxss)
 
 		if (core->vfb &&
 			(format->vf_code != XVIP_VF_YUV_422) &&
+			(format->vf_code != XVIP_VF_YUV_420) &&
 			(format->vf_code != XVIP_VF_RBG) &&
 			(format->vf_code != XVIP_VF_MONO_SENSOR)) {
 			dev_err(core->dev, "Invalid UG934 video format set.\n");
-- 
2.31.1

